scripts/public-circleci-lib-dist.js

```diff
@@ -2,13 +2,36 @@
 const release = require('../utils/release');
 const fs = require('fs-extra');
 
+async function getDistFiles() {
+  const distfiles = [];
+
+  // Don't override the dist/package.json if it's already bundled.
+  if (!fs.existsSync('./dist/package.json')) {
+    distfiles.push(fs.copy('./package.json', './dist/package.json'));
+  }
+
+  if (fs.existsSync('./readme.md')) {
+    distfiles.push(fs.copy('./readme.md', './dist/readme.md'));
+  } else if (fs.existsSync('./README.md')) {
+    distfiles.push(fs.copy('./README.md', './dist/README.md'));
+  } else {
+    console.log('No readme/README found.');
+  }
+
+  if (fs.existsSync('./license')) {
+    distfiles.push(fs.copy('./license', './dist/license'));
+  } else if (fs.existsSync('./LICENSE')) {
+    distfiles.push(fs.copy('./LICENSE', './dist/LICENSE'));
+  } else {
+    console.log('No license/LICENSE found.');
+  }
+
+  return distfiles;
+}
+
```

Comment:
```suggestion
async function copyBestPath(bestPaths, targetPath) {
  const bestPath = bestPaths.find((potentialBestPath) => {
    return fs.existsSync(potentialBestPath)
  })
  if (bestPath === undefined) {
    console.log(`Could not find a best path for ${targetPath} from ${bestPaths.join(' ')}`)
  }
  return fs.copy(bestPath, targetPath)
}
```

Reply:
Implemented feedback in the latest commit. Thank you again @ryansmith94